mzLib/Proteomics/RetentionTimePrediction/Chronologer/Chronologer.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.IO;
+using TorchSharp;
+using TorchSharp.Modules;
+
+namespace Proteomics.RetentionTimePrediction.Chronologer
+{
+    /// <summary>
+    /// Chronologer is a deep learning model for highly accurate prediction of peptide C18 retention times (reported in % ACN).
+    /// Chronologer was trained on a new large harmonized database of > 2.6 million retention time observations
+    /// (2.25M unique peptides) constructed from 11 community datasets
+    /// and natively supports prediction of 17 different modification types.
+    /// With only a few observations of a new modification type (> 10 peptides),
+    /// Chronologer can be easily re-trained to predict up to 10 user supplied modifications.
+    ///
+    /// Damien Beau Wilburn, Ariana E. Shannon, Vic Spicer, Alicia L. Richards, Darien Yeung, Danielle L. Swaney, Oleg V. Krokhin, Brian C. Searle
+    /// bioRxiv 2023.05.30.542978; doi: https://doi.org/10.1101/2023.05.30.542978
+    /// 
+    /// https://github.com/searlelab/chronologer
+    ///
+    /// Licensed under Apache License 2.0
+    /// 
+    /// </summary>
+    internal class Chronologer : torch.nn.Module<torch.Tensor, torch.Tensor>
+    {
+        public Chronologer() : this(Path.Combine(AppDomain.CurrentDomain.BaseDirectory,
```

Comment:
The class is internal, but a lot of methods are public. I'd suggest keeping the class and constructor protection levels consistent